src/main/cocoa/CocoaCairoSurface.mm

```diff
@@ -62,8 +62,8 @@ static inline cairo_antialias_t decode_antialiasing(const Font &f)
             {
                 pDisplay        = dpy;
                 pCocoaWindow    = window;
-                fWidth          = width;
-                fHeight         = height;
+                nWidth          = width;
```

Comment:
You don't need to initialize `nWidth` and `nHeight` here as they are initialized in constructor of `ws::ISurface`.